docs/user_manual/source/pipeline.rst

```diff
@@ -68,11 +65,17 @@ and zero stall on the data-side memory interface.
   |                       | 35 (cpuctrl.dataindtiming is set)    | The maximum number of cycles is 35 when the divider is 0    |
   |                       |                                      |                                                             |
   +-----------------------+--------------------------------------+-------------------------------------------------------------+
-  | Jump                  | 3                                    | Jumps are performed in the ID stage. Upon a jump the IF     |
+  | Jump                  | 2 (PC hardening disabled)            | Jumps are performed in the ID stage. Upon a jump the IF     |
```

Comment:
Branch (not taken) does not seem correct; 1 cycle should be possible. Also update comments

Reply:
Fixed, split the untaken branches into enabled/disabled PC hardening.